### ISSUE TITLE
[BENCHMARK DATASET REQUEST] NetherlandsForensicInstitute/squad-nl-v2.0

### ISSUE THREAD

**Issue by saattrupdan** [ScandEval/ScandEval#670]
### Dataset name

NetherlandsForensicInstitute/squad-nl-v2.0

### Dataset link

https://huggingface.co/datasets/NetherlandsForensicInstitute/squad-nl-v2.0

### Dataset languages

- [ ] Danish
- [ ] Swedish
- [ ] Norwegian (Bokmål or Nynorsk)
- [ ] Icelandic
- [ ] Faroese
- [ ] German
- [X] Dutch
- [ ] English

### Describe the dataset

This is also a machine translated version of SQuAD to Dutch, but the translation is better than the current one we use, so this should simply replace the old dataset. Note that we also need to re-benchmark all the models on this new one.

See a comparison [here](https://huggingface.co/datasets/yhavinga/squad_v2_dutch/discussions/2#6763ed4c42436c7f7005f4b4).

**Comment by wietsedv** [comment]
Fyi; I am the creator of this version of SQuAD-NL and here are some extra references.

The paper where it was introduced can be found here: https://aclanthology.org/2023.emnlp-main.447/ Also, I have a leaderboard thingy here: https://dumbench.nl/ where models achieve F1 up to 84%. The NFI data upload on Hugging Face is identical to my version that I uploaded here: https://github.com/wietsedv/NLP-NL/tree/main

**Comment by Rijgersberg** [comment]
Yes, we just added an extra column to the dataset for the evaluation of embedding models. Better to create a new, canonical upload for the benchmark, I think. Huggingface Datasets makes this pretty easy.

The code used to make the NFI upload is provided in the bottom of the model card.

**Comment by wietsedv** [comment]
That is actually a very good point, thanks for pointing out that I have not done that yet. I uploaded the canonical version of SQuAD-NL v2.0 here: https://huggingface.co/datasets/GroNLP/squad-nl-v2.0 and v1.1 here: https://huggingface.co/datasets/GroNLP/squad-nl-v1.1

**Comment by Rijgersberg** [comment]
Thanks! I added a link for better attribution.
